django/core/management/commands/loaddata.py

```diff
@@ -43,6 +44,9 @@ def add_arguments(self, parser):
             'fixtures into. Defaults to the "default" database.')
         parser.add_argument('--app', action='store', dest='app_label',
             default=None, help='Only look for fixtures in the specified app.')
+        parser.add_argument('-e', '--exclude', dest='exclude', action='append', default=[],
```

Comment:
Unless I'm missing something, `dest='exclude'` is actually not needed, but I see that it's the common pattern in other files in `django/core/management/commands/`.